src/main/java/uk/gov/service/notify/NotificationClient.java

```diff
@@ -310,9 +311,21 @@ public static JSONObject prepareUpload(final byte[] documentContents) throws Not
 
         JSONObject jsonFileObject = new JSONObject();
         jsonFileObject.put("file", fileContent);
+        jsonFileObject.put("is_csv", isCsv);
         return jsonFileObject;
     }
 
+    /**
+     * Use the prepareUpload method when uploading a document via sendEmail.
+     * The prepareUpload method creates a <code>JSONObject</code> which will need to be added to the personalisation map.
+     *
+     * @param documentContents byte[] of the document
+     * @return <code>JSONObject</code> a json object to be added to the personalisation is returned
+     */
+    public static JSONObject prepareUpload(final byte[] documentContents) throws NotificationClientException {
+        return prepareUpload(documentContents, Boolean.FALSE);
```

Comment:
Any idea if I should generally be using `Boolean.FALSE` or if I should just use `false`?

Reply:
It doesn't matter - seems `false` has been used elsewhere

Reply:
It doesn't matter. Looks like `false` is used elsewhere.